tests/Fixer/FunctionNotation/FunctionTypehintSpaceFixerTest.php

```diff
@@ -56,54 +56,109 @@ public function provideFixCases()
             [
                 '<?php function foo(array & $param) {}',
                 '<?php function foo(array& $param) {}',
             ],
+            [
+                '<?php function foo(array & $param) {}',
+                '<?php function foo(array  & $param) {}',
+            ],
             [
                 '<?php function foo(Bar $param) {}',
                 '<?php function foo(Bar$param) {}',
             ],
+            [
+                '<?php function foo(Bar $param) {}',
+                '<?php function foo(Bar  $param) {}',
+            ],
             [
                 '<?php function foo(Bar\Baz $param) {}',
                 '<?php function foo(Bar\Baz$param) {}',
             ],
+            [
+                '<?php function foo(Bar\Baz $param) {}',
+                '<?php function foo(Bar\Baz  $param) {}',
+            ],
             [
                 '<?php function foo(Bar\Baz &$param) {}',
                 '<?php function foo(Bar\Baz&$param) {}',
             ],
+            [
+                '<?php function foo(Bar\Baz &$param) {}',
+                '<?php function foo(Bar\Baz  &$param) {}',
+            ],
             [
                 '<?php function foo(Bar\Baz & $param) {}',
                 '<?php function foo(Bar\Baz& $param) {}',
             ],
+            [
+                '<?php function foo(Bar\Baz & $param) {}',
+                '<?php function foo(Bar\Baz  & $param) {}',
+            ],
             [
                 '<?php $foo = function(Bar\Baz $param) {};',
                 '<?php $foo = function(Bar\Baz$param) {};',
             ],
+            [
+                '<?php $foo = function(Bar\Baz $param) {};',
+                '<?php $foo = function(Bar\Baz  $param) {};',
+            ],
             [
                 '<?php $foo = function(Bar\Baz &$param) {};',
                 '<?php $foo = function(Bar\Baz&$param) {};',
             ],
+            [
+                '<?php $foo = function(Bar\Baz &$param) {};',
+                '<?php $foo = function(Bar\Baz  &$param) {};',
+            ],
             [
                 '<?php $foo = function(Bar\Baz & $param) {};',
                 '<?php $foo = function(Bar\Baz& $param) {};',
             ],
+            [
+                '<?php $foo = function(Bar\Baz & $param) {};',
+                '<?php $foo = function(Bar\Baz  & $param) {};',
+            ],
             [
                 '<?php class Test { public function foo(Bar\Baz $param) {} }',
                 '<?php class Test { public function foo(Bar\Baz$param) {} }',
             ],
+            [
+                '<?php class Test { public function foo(Bar\Baz $param) {} }',
+                '<?php class Test { public function foo(Bar\Baz  $param) {} }',
+            ],
             [
                 '<?php $foo = function(array $a,
-                    array $b, array     $c, array
-                    $d) {};',
+                    array $b, array $c, array $d) {};',
```

Comment:
looking nice

Reply:
What do you mean, is something wrong?

Reply:
i tried to show i was happy with change i saw in diff ;)

Reply:
Ha, nice! 👍